api/spec/controllers/spree/api/option_values_controller_spec.rb

```diff
@@ -49,7 +49,7 @@ def check_option_values(option_values)
 
       it "can retrieve a list of option types" do
         option_value_1 = create(:option_value, option_type: option_type)
-        option_value_2 = create(:option_value, option_type: option_type)
+        create(:option_value, option_type: option_type)
```

Comment:
Same here, I would rewrite this with a `context`, a `before` block and a `let!(:option_value_1)`